### ISSUE TITLE
[Bug][a11y]Talkback jumps from reading the Tabs to search bar 

### ISSUE THREAD

**Issue by sv-sdeiac** [mozilla-mobile/fenix#4489]
## Prerequisites
You have Talkback activated on your device.

## Steps to reproduce
1. Open Firefox Preview;
2. Open a page;
3. Open tab drawer;
4. Quick after opening tab drawer, tap on your tab from Open tabs section;

### Expected behavior
Talkback read the tab;

### Actual behavior
Talkback starts to read the tab and after that jump on the search bar;

### Device information

* Android device: Nexus 9 (Android 7.1.1), Motorola G4 (Android 6.0.1).
* Fenix version: 1.2.0-rc.2.

### Notes
- It seems that if you come from tab drawer the focus of Talkback is on the Search bar and is announced 2 times.

**Comment by ekager** [comment]
I undid the accessibility changes from #2142 so this should be resolved now, closing
